pom.xml

```diff
@@ -56,54 +56,54 @@
         <dependency>
             <groupId>org.apache.maven.plugin-tools</groupId>
             <artifactId>maven-plugin-annotations</artifactId>
-            <version>3.5</version>
+            <version>3.6.0</version>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>org.codehaus.plexus</groupId>
             <artifactId>plexus-utils</artifactId>
-            <version>3.0.24</version>
+            <version>3.1.0</version>
         </dependency>
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-plugin-api</artifactId>
-            <version>3.3.9</version>
+            <version>3.6.0</version>
         </dependency>
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-core</artifactId>
-            <version>3.3.9</version>
+            <version>3.6.0</version>
```

Comment:
Does this bump the minimum support version of Maven to 3.6 as well?  I don't _think_ it should, but I guess we should check.

Reply:
Fair question, I'll see what I can find...

Reply:
As an anecdotal data point, my workstation is using Maven 3.5.0, and I'm able to build the plugin and use the SNAPSHOT version successfully. That's not a general answer, though. I'll keep looking.

Reply:
I'm asking the internet - https://stackoverflow.com/q/53876133/587091.

Reply:
I suppose if it causes an issue, we can always drop it back down.